codes.nora.gDiceRoller.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "codes.nora.gDiceRoller",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "codes.nora.gDiceRoller",
+    "finish-args" : [
+        "--socket=x11",
```

Comment:
you should also add --share=ipc per http://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
Will do, thanks so much for your help!